Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -349,6 +349,26 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
         Add-AnalyzedResultInformation @params
     }
 
+    if ($null -ne $exchangeInformation.TransportService) {
+        if ($exchangeInformation.TransportService.MaxPerDomainOutboundConnections -lt 40) {
+            $params = $baseParams + @{
+                Name             = "MaxPerDomainOutboundConnections"
+                Details          = "Value set to $($exchangeInformation.TransportService.MaxPerDomainOutboundConnections), which is less than the recommended value of 40. `r`n`t`tMore details: https://aka.ms/TransportRetryConfig"
+                DisplayWriteType = "Yellow"
+            }
+            Add-AnalyzedResultInformation @params
+        }
+
+        if ($exchangeInformation.TransportService.MessageRetryInterval -gt [System.TimeSpan]::FromMinutes(5)) {
```

Comment:
I don't believe this is going to work with Remote PowerShell, as we don't normally have data types there. Please verify this.

Reply:
I see many other data types referenced in the script (version, datetime, etc). I can verify though. What does remote PowerShell mean in this context? New-PsSession? Invoke-Command?

Reply:
https://learn.microsoft.com/en-us/powershell/exchange/connect-to-exchange-servers-using-remote-powershell?view=exchange-ps#connect-to-a-remote-exchange-server

These are the steps.

Reply:
Confirmed this datatype works with remote powershell session (New-PsSession).